components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -346,9 +347,22 @@ class GeckoEngine(
             } ?: emptyList()
 
             extensions.forEach { extension ->
+                // https://github.com/mozilla-mobile/android-components/issues/6227 set all installed
+                // extensions to be allowed in private browsing mode. We need to revert back all the
```

Comment:
nit: Two "back"s. We need to revert back to false which is now the default.... 

or similar.

Reply:
Unresolved this as the change didn't get applied.